### ISSUE TITLE
Incorrect battery measurements

### ISSUE THREAD

**Issue by adamgarbo** [sparkfun/MicroMod_Data_Logging_Carrier#4]
Hi folks,

I noticed that I was getting incorrect battery voltage measurements when powering the MMDLCB with a LiPo battery. After looking at the schematic, this seems to be due to the fact that VBAT has to pass through D3 before getting to the top of the voltage divider. With a multimeter, I measured a battery voltage of 4.2 V but only 4.1 V at the top of the voltage divider, meaning there's a voltage drop of ~0.1 V across D3.

![image](https://user-images.githubusercontent.com/22924092/107982716-8d368180-6f92-11eb-8ab1-329c4080cb74.png)

This looks to apply to all MicroMod carrier boards with the onboard resistor divider. While it can be addressed using a correction factor in the code, it may be a good idea to add it to the list for the next hardware revision of the MMDLCB.

TL;DR: Feed onboard voltage divider with VBAT and not VIN

Cheers,
Adam
